tests/app/views/handlers/test_individual_response_fulfilment_request.py

```diff
@@ -87,19 +71,31 @@ def test_postal_fulfilment_request_payload():
             }
         },
     }
-    assert postal_json_payload == expected_sms_payload
+    assert postal_json_message == expected_sms_message
+
+
+def validate_uuids_in_message(json_message):
+    transaction_id = json_message["event"].pop("transactionId")
```

Comment:
Testing of the base abstract class should be done in a separate test, as it applies to any concrete instance of the base class. A "fake" concrete instance can be instantiated in the test for this, much like we do in the Form testing.

Reply:
My bad should have done that in the first place. Have updated accordingly.